openshift/postgresql.dc.json

```diff
@@ -359,7 +252,7 @@
                     "valueFrom": {
                       "secretKeyRef": {
                         "key": "oracle-fdw-name",
-                        "name": "${DATABASE_SERVICE_NAME}"
+                        "name": "${FDW_SECRET_NAME}"
```

Comment:
Since this variable is `FDW_NAME`, having the secret key reference be named `FDW_SECRET_NAME` might cause some confusion (in that it references the name of the secret, not the FDW name). As `FDW_SECRET_NAME` contains other values like the schema name and password, suggest that it be renamed to `FDW_SECRET`

Reply:
Oh I see the confusion. But the FDW name is actually coming in from the secret.

So we go to the secret and fetch the name from in there and pass it to the env variable FDW_NAME that goes to the container. 
We're not passing the FDW_NAME in the template, but fetching it from the secret